### DISCUSSION TITLE
Use personal CNAME domains instead of raw IPs

### DISCUSSION THREAD

**Discussion by cofob** [deta/cloud-docs#293] [Ideas] [answered]
At the moment to connect your own domain to Micro you need to add A record. This has several disadvantages, for example:
1. It is not possible to scale the service. If a client uses a lot of resources of one node, it is impossible to simply allocate him another address on a free node.
2. Binding to an IP address which can change. And if it changes, all clients will have to change their domains configuration.

This can be solved by issuing personal domains for each Micro. For example, in domain connection dialog display 
![](https://l.frsqr.xyz/piyzt)
Besides, the required functionality is present in deta.dev subdomains.

**Comment by aavshr** [comment, marked as answer]
Hello @cofob , thanks for the suggestions

> It is not possible to scale the service. If a client uses a lot of resources of one node, it is impossible to simply allocate him another address on a free node.

The IP addresses we provide are anycast IPs and not bounded to a single node/server. 

> Binding to an IP address which can change. And if it changes, all clients will have to change their domains configuration.

The IP addresses we provide are static IPs; they will not change on their own. If we have to change them for some reason, we will let the users know in time before hand and we'll keep the IPs active for a long enough time for people to migrate.

The `CNAME` suggestion still makes sense and we'll think about supporting it.
